.github/workflows/self-pr-slow-ci.yml

```diff
@@ -4,6 +4,11 @@ on:
   pull_request:
     paths:
       - "src/transformers/models/*/modeling_*.py"
+      - "tests/models/*/test_*.py"
```

Comment:
If nothing changed in those paths, the CI won't be triggered at all
(even if the commit message specifies some models)

Reply:
Yes, we will see 🚀 !